src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.tsx

```diff
@@ -34,13 +44,53 @@ export const RespondentDataReview = () => {
   const [isLastVersion, setIsLastVersion] = useState<boolean>(false);
   const [isBannerVisible, setIsBannerVisible] = useState<boolean>(false);
   const [itemIds, setItemIds] = useState<string[]>([]);
+  const [responseDates, setResponseDates] = useState<Date[]>();
+  const [activities, setActivities] = useState<ReviewActivity[]>([]);
   const [isLoading, setIsLoading] = useState(false);
-  const dataTestid = 'respondents-review';
-
   const [activityItemAnswers, setActivityItemAnswers] = useState<
     DecryptedActivityData<EncryptedAnswerSharedProps>['decryptedAnswers'] | null
   >(null);
 
+  const { control } = useForm({
```

Comment:
Could you pls rename the task M2-4485 and describe all changes that is made with components?
Because QA team need to know, that it's not only tests, but also changes in components, which may cause regression.
Pls make changes (refactoring) in components in different than unit tests tasks in the future (divide tasks to changes (refactoring code) and unit tests).